src/planner/parabolic/ParabolicSmoother.cpp

```diff
@@ -156,10 +156,49 @@ std::unique_ptr<aikido::trajectory::Spline> ParabolicSmoother::postprocess(
   auto timedTrajectory = computeParabolicTiming(
       *_inputTraj, mVelocityLimits, mAccelerationLimits);
 
+  auto shortcutOrBlendTrajectory
+      = handleShortcutOrBlend(std::move(timedTrajectory), _rng);
+  if (shortcutOrBlendTrajectory)
+    return shortcutOrBlendTrajectory;
+
+  return timedTrajectory;
+}
+
+//==============================================================================
+std::unique_ptr<aikido::trajectory::Spline> ParabolicSmoother::postprocess(
+    const std::unique_ptr<trajectory::Spline>& _inputTraj,
+    const aikido::common::RNG* _rng)
+{
+  if (!_rng)
```

Comment:
Could we change the type of `_rng` to const reference if we throw an exception for `nullptr` anyway?